Core/Registry/RegistryManager.cs

```diff
@@ -40,14 +40,24 @@ public class RegistryManager : IDisposable
         /// </summary>
         public string previousCorruptedPath;
 
+        private static string InstanceRegistryPath(string path)
```

Comment:
Shouldn't that be `InstanceRegistryLockFilePath` or so? Under `InstanceRegistryPath` I understand `CKAN/registry.json`.